### DISCUSSION TITLE
Question about the icon-contract-2 icon: is the icon-contract-2 icon adequate?

### DISCUSSION THREAD

**Discussion by AlainRnet** [FLEXIcontent/flexicontent-cck#1212] [General]
In an item (article), click on the "basket" icon (icon-contract-2) to compare 2 versions. When there are several pages of versions and we changes page, icons are then images and the basket becomes a magnifying glass.
I see that the icon appears to be built by https://github.com/FLEXIcontent/flexicontent-cck/blob/master/admin/views/item/tmpl/common/load_config_n_acl.php#L172 and that the image comes from https://github.com/FLEXIcontent/flexicontent-cck/blob/master/admin/controllers/items.raw.php#L85

Question based on this observation:
- Is this difference desired?
- Is the icon of the basket (icon-contract-2) used elsewhere?
- Otherwise, can we fix https://github.com/FLEXIcontent/flexicontent-cck/blob/master/admin/views/item/tmpl/common/load_config_n_acl.php#L172 and replace icon-contract-2 with search?

**Comment by micker** [comment]
Please post some screen

**Comment by micker** [comment]
i propose this
![image](https://github.com/user-attachments/assets/bf1c7539-c818-447e-bf5f-907a1037d87f)

**Comment by AlainRnet** [comment]
and this for the image?
![image](https://github.com/user-attachments/assets/09ccd7c8-9e45-491c-ae3d-575eb5cd886d)
img src="/administrator/components/com_flexicontent/assets/images/eye.png"

**Comment by micker** [comment]
done in master
